react/components/StepsBar/Step.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { scrollStep, roundedRight, roundedLeft } from './styles.css'
+
+const COMPLETED = 'completed'
+const IN_PROGRESS = 'inProgress'
+const TO_DO = 'toDo'
+
+class Step extends Component {
+  render() {
+    const { type, roundLeft, roundRight } = this.props
+    const background = {
+      background:
+        'repeating-linear-gradient(\n  -45deg,\n  #134CD8,\n  #134CD8 10px,\n  #89aeff 10px,\n  #89aeff 20px\n)',
+    }
+    const roundLeftCSS = roundLeft ? roundedLeft : ''
+    const roundRightCSS = roundRight ? roundedRight : ''
```

Comment:
since both these vars are always used in tandem, you could join both in a single, for example, `roundingStyle` variable